ui/page/overview/overview_layout.go

```diff
@@ -372,13 +389,46 @@ func (pg *AppOverviewPage) listenForSyncNotifications() {
 				case wallet.SyncStarted:
 					fallthrough
 				case wallet.SyncCanceled:
-					fallthrough
+					pg.loadTransactions()
+					pg.RefreshWindow()
```

Comment:
Why not `fallthrough`? It's one line shorter and maintains the `fallthrough` sequence from the previous case to the last.